pancancer_utilities/tcga_utilities.py

```diff
@@ -0,0 +1,180 @@
+"""
+Functions for preprocessing TCGA expression data and mutation status labels.
+
+Most functions are adapted from:
+https://github.com/greenelab/BioBombe/blob/master/9.tcga-classify/scripts/tcga_util.py
+"""
+import os
+
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+
+def process_y_matrix(
+    y_mutation,
+    y_copy,
+    include_copy,
+    gene,
+    sample_freeze,
+    mutation_burden,
+    filter_count,
+    filter_prop,
+    output_directory,
+    hyper_filter=5,
+):
+    """
+    Combine copy number and mutation data and filter cancer-types to build y matrix
+
+    Arguments
+    ---------
+    y_mutation: Pandas DataFrame of mutation status
+    y_copy: Pandas DataFrame of copy number status
+    include_copy: boolean if the copy number data should be included in status calc
+    gene: string indicating gene of interest (used for writing proportion file)
+    sample_freeze: pandas dataframe storing which samples to use
+    mutation_burden: pandas dataframe storing log10 mutation counts
```

Comment:
Just checking my understanding. So mutation burden and copy status are perform at a sample-level where mutation burden indicates how many mutations compared to some reference and copy status is how many copies of a gene respectively?

Reply:
Yep, exactly!